zulipterminal/ui_tools/buttons.py

```diff
@@ -82,6 +83,9 @@ def update_widget(
         self.button_suffix.set_text(suffix)
         self._w.set_attr_map({None: text_color})
 
+    def update_widget_highlight(self, highlight_style: str) -> None:
+        self.original_color = highlight_style
```

Comment:
In WSL, it did not highlight active stream until I added line ``self._w.set_attr_map({None: highlight_style})`` as suggested by ``Neil``.